Mathlib/Data/Matrix/PEquiv.lean

```diff
@@ -61,7 +61,7 @@ theorem toMatrix_apply [DecidableEq n] [Zero α] [One α] (f : m ≃. n) (i j) :
 #align pequiv.to_matrix_apply PEquiv.toMatrix_apply
 
 theorem mul_matrix_apply [Fintype m] [DecidableEq m] [Semiring α] (f : l ≃. m) (M : Matrix m n α)
-    (i j) : (f.toMatrix ⬝ M) i j = Option.casesOn (f i) 0 fun fi => M fi j := by
+    (i j) : (f.toMatrix * M :) i j = Option.casesOn (f i) 0 fun fi => M fi j := by
```

Comment:
Without the colon lean complains that it can't work out the motive for the eliminator or something. This is not the same problem as the other one, but perhaps is worth drawing attention to.